tsconfig.json

```diff
@@ -42,8 +46,9 @@
 			"@woocommerce/knobs": [ "storybook/knobs" ],
 			"@woocommerce/settings": [ "assets/js/settings/shared" ],
 			"@woocommerce/shared-context": [ "assets/js/shared/context" ],
-			"@woocommerce/type-defs/*": [ "assets/js/type-defs/*" ]
+			"@woocommerce/type-defs/*": [ "assets/js/type-defs/*" ],
+			"@woocommerce/types": [ "assets/js/type-defs" ]
 		}
 	},
-	"include": [ "./assets/js/**/*.js", "./globals.d.ts" ]
+	"include": [ "./assets/js/**/*.js", "./globals.d.ts", "./assets/js/**/*.ts" ]
```

Comment:
Do we want to remove `.js` from here? It seems with the addition of the `no-implicit-any` eslint rule, the checking is kind of aggressive and in my IDE at least it just paints entire .js files red.

Reply:
VScode has jsdoc typescript support so I don't get the same behaviour as you in PHPStorm. We have `.js` here to get that implicit type-checking through the jsdocs on the js files. Maybe there's something that could be setup in PHPStorm to improve that?